pyfuncol/extend_builtins.py

```diff
@@ -0,0 +1,145 @@
+from warnings import warn
+
+from . import dict as pfcdict
+from . import list as pfclist
+from . import set as pfcset
+
+EXTEND_BUILTINS = False
+try:
+    from forbiddenfruit import curse
+    EXTEND_BUILTINS = True
+except ImportError:
+    warn("[WARNING] You are using pyfuncol without forbiddenfruit. Functions will not be extended to builtins")
```

Comment:
```suggestion
    warn("[WARNING] You are using pyfuncol without forbiddenfruit. Functions will not extend builtins")
```

Reply:
Actually, shouldn't it be: "functions will not be extended to builtins" because the builtins are being extending with these new functions? Will commit anyway.

Reply:
As it is a small detail, we merge anyway :)